cmake/modules/AddSwift.cmake

```diff
@@ -465,6 +465,10 @@ function(_add_swift_lipo_target)
   is_darwin_based_sdk("${LIPO_SDK}" IS_DARWIN)
   if(IS_DARWIN)
     if(LIPO_CODESIGN)
+      if(NOT CMAKE_CODESIGN)
+        message(FATAL_ERROR "CMAKE_CODESIGN is required, please set it")
```

Comment:
Could use `precondition(CMAKE_CODESIGN MESSAGE "CMAKE_CODESIGN is required, please set it")